packages/playground-server/src/resources/user/resource.ts

```diff
@@ -1,6 +1,7 @@
 import { Resource } from "@ebryn/jsonapi-ts";
 
 export default class User extends Resource {
+  // @ts-ignore
```

Comment:
Why add this?

Reply:
Otherwise, the linter throws an error here because `attributes` isn't initialized. Not sure why this isn't failing on master.